modules/managed-cluster-add-notification-contacts.adoc

```diff
@@ -27,3 +27,11 @@ By default, only the cluster owner receives cluster notification emails. You can
 
 .Verification steps
 * The "Notification contact added successfully" message displays.
+
+.Troubleshooting
+
+Error: `Account with ID <username> denied access to perform create on NotificationContact`::
```

Comment:
Hi @laubai  , The error message never indicated the user name but the account id belongs to user will show.
Btw, This error currently happens only when user granted with role "Cluster autoscaler editor" now and it is bug tracked under [OCMUI-729](https://issues.redhat.com/browse/OCMUI-729?focusedId=22947967&page=com.atlassian.jira.plugin.system.issuetabpanels:comment-tabpanel#comment-22947967)  . For all other unsupported user roles , the notification option will be disabled with tool tip as below.
![Screenshot from 2024-09-03 12-16-41](https://github.com/user-attachments/assets/d0702807-203b-4d53-8b3c-21989f872ae4)

Hence not sure if we want to include the above definition as part of the doc definition. Please review it. Thanks

Reply:
Done, thanks for the heads up on that!